go-controller/pkg/ovn/egressgw.go

```diff
@@ -344,6 +344,38 @@ func (oc *Controller) deleteLogicalRouterStaticRoute(podIP, mask, gw, gr string)
 	return nil
 }
 
+// deleteOnePodGWRoute deletes all associated gateway routing resources for one
+// pod gateway route
+func (oc *Controller) deleteOnePodGWRoute(routeInfo *externalRouteInfo, podIP, gw, gr string) {
```

Comment:
do we really need the word "One" in the function name. Doesn't the singular use of Route imply its just one?

Reply:
Removed.